vendor/usexp/src/usexp.ml

```diff
@@ -64,17 +64,21 @@ end
 
 type t =
   | Atom of string
+  | String of string
   | List of t list
 
 type sexp = t
 
 let rec to_string = function
   | Atom s -> Atom.serialize s
+  | String s -> Atom.serialize s
```

Comment:
`Atom.serialize` will only add the double quotes if necessary. In this case, we should add them systematically

Reply:
Done.